cluster-autoscaler/go.sum

```diff
@@ -103,6 +103,8 @@ github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751/go.mod h1:LOuy
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d/go.mod h1:rBZYJk541a8SKzHPHnH3zbiI+7dagKZ0cgpgrD7Fyho=
+github.com/aliyun/alibaba-cloud-sdk-go v1.61.62 h1:APZ+/l4TsHbFx6zZnXJE2QCXs3IChVmcI5GKwZnnmaw=
```

Comment:
The purpose of embedding sdk in cloud provider folder is to not add cloud provider specific external dependencies to CA. Please remove these lines.

Reply:
Thanks for reviewing! The new alibaba sdk  need the dependency,  gopkg.in/ini.v1 which  is not in vendor yet. Shoud I keep the dependency information in go.sum, go.mod and vendor/modules.txt or remove it?

Reply:
I'm sorry but with 30 cloud providers around, to keep our integrity, we cannot make exceptions. There should be no additional dependencies added for any of the cloud providers, neither in go.mod or vendor, no matter how harmless the dependency may look like.